src/averageAges.js

```diff
@@ -15,11 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((person) => {
+    return century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm';
+  }
+  );
+
+  const menAge = men.map((man) => {
+    return {
+      age: man.died - man.born,
+    };
+  });
+
+  const totalAge = menAge.reduce((sum, { age }) => sum + age, 0);
```

Comment:
there's no need to create separate variable, just put condition inside reduce method